CHANGELOG.rst

```diff
@@ -4,7 +4,8 @@ CHANGELOG
 5.3.63 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- Update Transaction.contains, Transaction.tpc_commit
```

Comment:
Can you describe what you're updating here? That contains is now checking the current app txn and the commit ordering has changed?